src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -1227,6 +1228,16 @@ private void recallClicked()
         MineColonies.getNetwork().sendToServer(new RecallTownhallMessage(townHall));
     }
 
+    /**
+     * Action when the hire button is clicked
+     */
+    private void hireClicked()
+    {
+        Log.getLogger().warn("hire clicked");
```

Comment:
Not sure if we need that?

Reply:
oh good catch thats and old log message gonna remove it tomorrow